jsons/UnitPromotions.json

```
@@ -812,4 +812,29 @@
 		"name": "[Royal Cavalry] ability",
         "uniques": ["[+100]% Strength <when attacking>"]
 	},
-]
\ No newline at end of file
+	// Ability Card Uniques
+	{
+		"name": "Screen Border Card",
+		"uniques": ["May Paradrop up to [100] tiles from inside friendly territory <for [Scout] units> <with [Screen Border Card]>"]
```

I find this too OP since scout units can basically paradrop to every corner of the map for the rest of the game, and Yukari's phantasm promotion already gives them a balanced 4 paradrop range anyway (which I may change to 5)
How about `"May enter foreign tiles without open borders <when not at war>"`?
```suggestion
		"uniques": ["May enter foreign tiles without open borders <when not at war> <for [Scout] units> <with [Screen Border Card]>"]
```

---

I don't think that's the OP, but I think there's a problem. This is because units can blockade resources on the sea.

---

How About May withdraw before melee ([32]%) <for [Melee] units>?

---

Right, that would do

---

Also, I forgot that they can only paradrop when in friendly tiles lol, perhaps the last one would still do, but make it `"May Paradrop up to [100] tiles from inside friendly territory <if this unit hasn't used other actions yet> <for [Scout] units> <with [Screen Border Card]>"` 
Just like in the Phantasm promotion, it would give units some kind of a delay

---

wow It's Amazing